it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -190,229 +190,229 @@ sendAnyway=Send anyway
-13000=There was an error copying the message to the Drafts folder. Retry?
+13000=Si è verificato un errore durante la copia del messaggio nella cartella Bozze. Riprovare?
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_ENCRYPT_TO_PLAIN_NO_SSL
 # LOCALIZATION NOTE (12594): $S is server hostname
-12594=There was an error copying the message to the Drafts folder. Retry?
+12594=Si è verificato un errore durante la copia del messaggio nella cartella Bozze. Riprovare?
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_ENCRYPT_TO_PLAIN_SSL
 # LOCALIZATION NOTE (12595): $S is server hostname
-12595=The SMTP server %S does not seem to support encrypted passwords. If you just set up this account, please try changing to 'Normal password' as the 'Authentication method' in the 'Account Settings | Server settings'. If it used to work and now suddenly fails, please contact your email administrator or provider.
+12595=Sembra che il server SMTP %S non supporti le password crittografate. Se stai impostando questo account, prova a impostare 'Password normale' come 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'. Se prima funzionava ma improvvisamente ora non funziona, contatta il tuo amministratore o fornitore e-mail.
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_PLAIN_TO_ENCRYPT
 # LOCALIZATION NOTE (12596): $S is server hostname
-12596=The SMTP server %S does not allow plaintext passwords. Please try changing to 'Encrypted password' as the 'Authentication method' in the 'Account Settings | Server settings'.
+12596=Il server SMTP %S non consente password con testo semplice. Prova a cambiare in 'Password crittografate' come 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'.
 
 ## @name NS_ERROR_SMTP_AUTH_FAILURE
 # LOCALIZATION NOTE (12597): $S is server hostname
-12597=Unable to authenticate to SMTP server %S. Please check the password, and verify the 'Authentication method' in 'Account Settings | Outgoing server (SMTP)'.
+12597=Impossibile autenticarsi al server SMTP %S. Controlla la password e verifica il 'Metodo di autenticazione' nelle 'Impostazioni account | Server in uscita (SMTP)'.
 
 ## @name NS_ERROR_SMTP_AUTH_GSSAPI
 # LOCALIZATION NOTE (12598): $S is server hostname
-12598=The Kerberos/GSSAPI ticket was not accepted by the SMTP server %S. Please check that you are logged in to the Kerberos/GSSAPI realm.
+12598=Il ticket Kerberos/GSSAPI non è stato accettato dal server SMTP %S. Controlla che hai effettuato l'accesso al reame Kerberos/GSSAPI.
 
 ## @name NS_ERROR_SMTP_AUTH_MECH_NOT_SUPPORTED
 # LOCALIZATION NOTE (12599): $S is server hostname
-12599=The SMTP server %S does not support the selected authentication method. Please change the 'Authentication method' in the 'Account Settings | Outgoing Server (SMTP)'.
+12599=Il server SMTP %S non supporta il metodo di autenticazione selezionato. Si prega di cambiare il 'Metodo di autenticazione' nelle 'Impostazioni account | Server in uscita (SMTP)'.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+12599=Il server SMTP %S non supporta il metodo di autenticazione selezionato. Modifica il 'Metodo di autenticazione' nelle 'Impostazioni account | Server in uscita (SMTP)'.